app/controllers/api/v2/site_statuses_controller.rb

```diff
@@ -17,7 +17,6 @@ def site_status_params
           .require(:site_status)
           .except(:id, :type, :site_id, :site_type, :has_vacancies?)
           .permit(
-            :applications_accepted_from,
```

Comment:
@dankmitchell assuming nope as frontend don't have seems to send that value along...